test/dynamo/test_trace_rules.py

```diff
@@ -169,6 +174,27 @@ def generate_allow_list():
     torch_object_ids = dict()
     torch_objects = set()
 
+    def heuristic_record_if_in_graph_function(obj, module, name):
```

Comment:
I have no idea what this does, so I suggest doc'ing it

Reply:
Will do